lib/AST/LocalizationFormat.cpp

```diff
@@ -121,6 +135,7 @@ YAMLLocalizationProducer::YAMLLocalizationProducer(llvm::StringRef filePath) {
   llvm::MemoryBuffer *document = FileBufOrErr->get();
   diag::LocalizationInput yin(document->getBuffer());
   yin >> diagnostics;
+  removedDiagnostic = std::move(yin.removedDiagnosticsIDs);
```

Comment:
@xedin This is a valid situation to use `std::move`, right?

Reply:
Yes, that's a valid use.